managers/views.py

```diff
@@ -68,13 +76,19 @@ class ProjectCreateView(CreateView):
     template_name = "managers/project_form.html"
 
     def get_context_data(self, **kwargs: Any) -> dict:
+        logger.info(f"User: {self.request.user} is in project create view")
         context_data = super().get_context_data(**kwargs)
         context_data["back_url"] = self.get_success_url()
         return context_data
 
     def get_success_url(self) -> str:  # pylint: disable=no-self-use
         return reverse("custom-projects-list")
 
+    def form_valid(self, form: ProjectForm) -> HttpRequest:
+        project = form.save()
```

Comment:
Isn't this being done `super().form_valid(form)` or somewhere else already?

Reply:
That's true, but how then I could log information about created project and his id ?